tests/reftests/working-dir.test

```diff
@@ -204,10 +204,9 @@ opam-version: "2.0"
 ### <pin:ongoing/ongoing.opam>
 opam-version: "2.0"
 build: [[ "test" "-f" "newfile.txt" ] [ "cat" "newfile.txt" ]]
-### opam install qux ongoing --working-dir -v | '.*(/|\\|")test(\.exe)?"? "' -> 'test "' | '.*(/|\\|")cat(\.exe)?"? "' -> 'cat "' | grep -v "switch import"
+### opam install qux ongoing --working-dir -v | '^Processing .+$' -> '\c' | '^-> retrieved .+$' -> '\c' | '.*(/|\\|")test(\.exe)?"? "' -> 'test "' | '.*(/|\\|")cat(\.exe)?"? "' -> 'cat "' | grep -v "switch import"
```

Comment:
Why retrieved too? There is only one that is done, for `qux`

Reply:
Because retrieving `qux` is done at the same time as building `ongoing` so the `-> retrieved qux.4` line can be before or after `test -f newfile.txt`

Reply:
As lines does not change content (as processiong) but only order, we can use `unordered`

```suggestion
### opam install qux ongoing --working-dir -v | '^Processing .+$' -> '\c' | '.*(/|\\|")test(\.exe)?"? "' -> 'test "' | '.*(/|\\|")cat(\.exe)?"? "' -> 'cat "' | grep -v "switch import" | unordered
```

Reply:
wouldn't `unordered` possibly change any line though?